### PR TITLE
Improve RuboCop setup

### PR DESCRIPTION
**Pull request by mokagio**
### Description

Nothing but some RuboCop-guided tweaks.

### Testing instructions

Nothing to test per se, but worth checking the diff just in case something broken syntax wise.

### Screenshots

N.A.
---

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. — N.A.

### CODE REVIEW

File: fastlane/Deliverfile
Comment (mokagio) [inline fastlane/Deliverfile]: Really, it's about time we remove the `deliver` setup from this repo, like we did in WordPress.

File: fastlane/Fastfile
Comment (mokagio) [inline fastlane/Fastfile:139]: These were actually unused.

**Comment by mokagio** [comment]
@crazytonyli for some reason I can't find you in the reviewers box, so here's a direct ping. Thank you for your time 🙇‍♂️

**Review by crazytonyli — APPROVED** [review]

**Comment by crazytonyli** [comment]
I think I can't be a reviewer because I am not in a group with write access. I can't merge this PR too.

@jkmassel Do you mind having a look if I'm in the right groups of this GitHub org?

**Comment by mokagio** [comment]
Not sure why auto-merge hasn't kicked in 🤔 The PR has been approved and CI is green...

image

I'll disable and merge manually.

---

Update: Just had to look closer... 👀 

The problem was with @crazytonyli's user access to the org, which is why the tick which usually is green is now green.

**Review by crazytonyli — APPROVED** [review]

**Comment by mokagio** [comment]
Still not working, grey tick...

image

But @crazytonyli, you are a member of the team...

image

**Comment by mokagio** [comment]
Ah! This was the issues I think... The new team had "Read" access to the repo, which was not enough to approve PRs:

image

**Pull request merged**
